types/Decimal.php

```diff
@@ -23,7 +23,8 @@ class Decimal extends AbstractMultiBaseType
         'thousands' => "\xE2\x80\xAF", // narrow no-break space
         'trimzeros' => true,
         'prefix' => '',
-        'postfix' => ''
+        'postfix' => '',
+		'engineering' => false
```

Comment:
indentation is off, please use 4 spaces not tabs

Reply:
Sorry, this is my first Pull Request on GitHub. Is the tab stop already fixed by you? And unfortunately I'm not familiar with unit tests on PHP.